rbac/core/kafka.py

```diff
@@ -35,7 +35,7 @@ class RBACProducer:
     def get_producer(self):
         """Init method to return fake kafka when flag is set to false."""
         if not hasattr(self, "producer"):
-            if settings.DEVELOPMENT or settings.MOCK_KAFKA or not settings.KAFKA_ENABLED:
+            if settings.MOCK_KAFKA or not settings.KAFKA_ENABLED:
```

Comment:
Do we not want to fake kafka if Development is true?

Reply:
Oh good eye, I forgot to remove - it was just for my debugging purposes. Thanks!